cmake/std/atdm/cee-rhel7/environment.sh

```diff
@@ -262,5 +262,8 @@ if [[ "${ATDM_CONFIG_SUPERLUDIST_INCLUDE_DIRS}" == "" ]] ; then
   export ATDM_CONFIG_SUPERLUDIST_LIBS=${SUPERLUDIST_ROOT}/lib64/libsuperlu_dist.a
 fi
 
+# Point CMake 3.19 compiler checks to missing symbols
+export LDFLAGS="$LDFLAGS -lifcore"
```

Comment:
This is only needed for intel builds. Other builds it causes an error as this library is not available in that environment. This one for example is clang:

https://testing.sandia.gov/cdash/build/8654297/configure

This one is GCC:

https://testing.sandia.gov/cdash/build/8654262/configure

-lifcore needs to be added individually to the 3 builds that were failing yesterday only.